packages/plugin-core/src/utils/editor.ts

```diff
@@ -236,3 +246,45 @@ export async function isBrokenWikilink(): Promise<boolean> {
   );
   return type === DECORATION_TYPES.brokenWikilink;
 }
+
+export type ProcessSelectionOpts = {
+  qs?: string;
+  vault?: DVault;
+  anchor?: DNoteAnchorBasic;
+  overrides?: Partial<NoteProps>;
+  kind?: TargetKind;
+  /**
+   * What {@link vscode.ViewColumn} to open note in
+   */
+  column?: ViewColumn;
+  /** added for contextual UI analytics. */
+  source?: string;
+};
+
+export async function getLinkFromSelectionWithWorkspace() {
```

Comment:
We discussed this offline, but for others - can you add a function header to clarify what 'WithWorkspace' means here?